Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocEnumValue.css

```diff
@@ -3,76 +3,80 @@
 span.bocEnumValue,
```

Comment:
hmm.. last tweak (I hope). I think we can drop the `div` and `span` selectors from .bocEnumValue in the file. this apears to be quite redundant.